README.md

```diff
@@ -110,76 +109,35 @@ npm run test:watch
-
-- I can click the "RSVP" button. When I do, I will be prompted to sign in. Then I will receive an email with a ticket and I will be added to the public list of event attendees.
-
-- I will receive a second email the day before the event to remind me.
-
-- After the event, I will automatically get emails notifying me of subsequent events.
-
-- I can filter all events in my location by tag/interests.
-
-### As an organizer
-
-- I can create a chapter.
-
-- I can edit details about the chapter, including a Slack/Discord/Facebook/WeChat/WhatsApp link participants can join to discuss and coordinate events.
-
-- I can create events, and set their location and capacity.
-
-- I can cancel events.
-
-- I can email the entire list of participants.
-
-- I can ban a participant whom I believe is toxic or who has previously broken my organization's code of conduct.
-
-- I can add a venue sponsor to the event with a link to their website as a way of thanking them for hosting.
-
-- I can add a food sponsor to the event with a link to their website as a way of thanking them for food.
-
-- I can see how many times a participant has come to the event as well as their attendance rate.
-
-- I can check-in attendees on the event registration desk with their `email_id` or `chapter_id`.
+1. Please check your idea does not already exist as an [MVP issue](https://github.com/freeCodeCamp/chapter/labels/MVP), or in the [post-MVP list](https://github.com/freeCodeCamp/chapter/issues/84).
+1. Create a new "Issue" and post details about the suggested user story.
 
 ## Roadmap
 
-1. Design the schema.
-2. Set up the API endpoints.
-3. Build the web client and let other developers use the API to build mobile clients and voice interface clients.
+The on-going [project Roadmap conversation](https://github.com/freeCodeCamp/chapter/issues/47) is regularly updated to reflect the overall progress and for higher-level discussions.
 
-Quincy Larson is the project lead. [FreeCodeCamp](https://www.freecodecamp.org) will start "dogfooding" this as soon as possible with several of its local study groups.
+Quincy Larson is the project lead. [FreeCodeCamp](https://www.freecodecamp.org) will start "dogfooding" the MVP with several of its local study groups.
 
 Here's an out-dated example of an app with similar functionality: [The freeCodeCamp Study Group Directory](https://study-group-directory.freecodecamp.org).
 
-You should [join our Discord server](https://discord.gg/vbRUYWS) to get connected with people interested in this project and to be aware of our future announcements.
 
 ## Contributing
 
-[**Take part in discussions or contribute code to this opensource codebase.**](CONTRIBUTING.md)
+* You should [join our Discord server](https://discord.gg/vbRUYWS) to get connected with people interested in this project and to be aware of our future announcements.
```

Comment:
Not required.

Reply:
This was previously in the README and I moved it. If we want to move it to CONTRIBUTING.md then that's fine with me.

Reply:
Discord is similarly mentioned in the CONTRIBUTING.md.

This note already existed in the README, so I have no strong feelings that it needs to be in the README.

However, if we leave it then I'd vote to move it out of the Roadmap section and into the Contributing section.

Reply:
@allella There's no harm in keeping it in readme's roadmap and contributing. Let it be.